.github/workflows/register_tre_bundle.yml

```diff
@@ -0,0 +1,270 @@
+---
+name: Register TRE Bundle
+
+# This workflow is intended to be used to register a TRE bundle into an existing TRE environment
+# by default, if not specified, the base workspace bundle will be registered into the TRE environment.
```

Comment:
wouldn't it be clearer to throw an error saying the argument is missing?

Reply:
I agree, done